srcpkgs/cni-plugins/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cni-plugins'
+pkgname=cni-plugins
+version=0.7.4
+revision=1
+wrksrc="plugins-${version}"
+build_style=go
+go_import_path="github.com/containernetworking/plugins"
+short_desc="Container Network Interface (plugins)"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/containernetworking/plugins"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
Please expand the homepage var here, it helps when the homepage needs to be changed.

Reply:
There are 14 of them, so it is certainly possible that new ones are added (or the existing ones re-ordered). I would say it is more likely than the configuration version changing.

Reply:
done.